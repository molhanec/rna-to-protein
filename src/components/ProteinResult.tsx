import React from 'react'
import { FormGroup, Label, Input } from 'reactstrap'

interface Props {
  protein: string
}

const ProteinResult = ({ protein }: Props) => (
  <FormGroup>
    <Label>
      Protein
    </Label>
    <Input readOnly value={protein} />
  </FormGroup>
)

export default ProteinResult
