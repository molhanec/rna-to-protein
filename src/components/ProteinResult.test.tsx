import React from 'react'
import { render } from '@testing-library/react'
import ProteinResult from './ProteinResult'

describe('<ProteinResult />', () => {
  it('matches snapshot', () => {
    const { container } = render(<ProteinResult protein="protein" />)
    expect(container).toMatchSnapshot()
  })
})
