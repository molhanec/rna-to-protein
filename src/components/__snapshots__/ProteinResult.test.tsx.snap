// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProteinResult /> matches snapshot 1`] = `
<div>
  <div
    class="form-group"
  >
    <label
      class=""
    >
      Protein
    </label>
    <input
      class="form-control"
      readonly=""
      type="text"
      value="protein"
    />
  </div>
</div>
`;
